core/variant_op.cpp

```diff
@@ -413,7 +413,7 @@ void Variant::evaluate(const Operator &p_op, const Variant &p_a,
 
 				_RETURN(p_a._data._bool == p_b._data._bool);
 			}
-
+			
```

Comment:
Trailing whitespace in various places in the file should be removed. (run it through clang-format to be sure)

Reply:
okay sure I can do that